src/idpyoidc/server/oauth2/token_helper.py

```diff
@@ -618,6 +619,25 @@ def _validate_configuration(self, config):
                 f"Unsupported default requested_token_type {_default_requested_token_type}"
             )
 
+    def get_handler_key(self, request, endpoint_context):
+        if "token_exchange" in endpoint_context.cdb[request["client_id"]]:
```

Comment:
OK

Reply:
@rohe why we have `default_requested_token_type` in both client_dict and self.config?

I'm worried about the possibility that one configures it in the general config but have it also in the client_dict and have forget this, and can't figure out why the parameter in the general config doesn't work

it would be better to have a single place of different names if there's the need to do overloading in spare places in the code/components